apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.TargetedSessionPool.cs

```diff
@@ -427,11 +427,14 @@ private async Task TryCreateReadWriteTransactionAndReturnToPool(PooledSession se
                 ReleaseInactiveSession(session, maybeCreateReadWriteTransaction: false);
             }
 
-            private async Task<PooledSession> BeginTransactionAsync(PooledSession session, TransactionOptions options, CancellationToken cancellationToken)
+            private async Task<PooledSession> BeginTransactionAsync(PooledSession session, TransactionOptions options, CancellationToken cancellationToken, bool isSessionAcquired = false)
```

Comment:
I don't fully understand what this parameter is yet. It's a little odd that nothing else has changed in the method, apart from whether or not the book-keeping. Maybe add a comment explaining this more?

Reply:
Yes, better thanks.